ajax/libs/amalthea/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "amalthea",
+  "version": "1.1.4",
+  "author": "Apollo <zhaoyang@jizhi.im>",
+  "description": "the hyper front-end engine of data science",
+  "keywords": [
+    "html-plugin",
+    "online-judge",
+    "jupyter-notebook"
+  ],
+  "homepage": "https://amalthea.ai",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Jinglue/Amalthea.git"
+  },
+  "npmName": "amalthea",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "amalthea.08fd947.bundle.js"
```

Comment:
We don't add hash code in filename. (If the newest version release the alto-update config can't get the files from new version because of the different filename)
Would it be revised to `amalthea.bundle.js` in future?
Or use `amalthea.js`?

PS. we fill in minified file's filename in this field. But these two files can be minify from other tool to minify in one line. This part maybe need to be discussed.

Many thanks.

Reply:
Thanks for your review and advice. I'll raise PR again when the library is more popular.

Sorry for disturbing.

Reply:
It's fine.
Still welcome to improve the open source project together.
Thank you for being interested in CDNJS. 😃